src/Cake/Infrastructure/ContainerBootstrapper.cs

```diff
@@ -0,0 +1,60 @@
+﻿using Cake.Client.Features.Building.Hosts;
+using Cake.Client.Infrastructure.Diagnostics;
+using Cake.Client.Infrastructure.Scripting;
+using Cake.Common.Modules;
+using Cake.Core;
+using Cake.Core.Composition;
+using Cake.Core.Configuration;
+using Cake.Core.Diagnostics;
+using Cake.Core.Modules;
+using Cake.Core.Scripting;
+using Cake.NuGet;
+using Spectre.Cli;
+
+namespace Cake.Client.Infrastructure
+{
+    /// <summary>
+    /// Responsible for registering all dependencies
+    /// that is required for executing a script.
+    /// </summary>
+    public interface IContainerBootstrapper
```

Comment:
New name for this? Don't like to use bootstrapper in another meaning than bootstrapping Cake scripts.

Reply:
I'm renaming this to `IContainerConfigurator`.